jaxrs2/src/main/resources/META-INF/beans.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://xmlns.jcp.org/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="
+        http://xmlns.jcp.org/xml/ns/javaee
+        http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd" bean-discovery-mode="all">
+</beans>
```

Comment:
This might sound overkill, but please provide a `jaxrs2` + `jaxrs2-cdi` module instead of `jaxrs2` with `beans.xml`. People having problems with CDI will have to unpack, remove the beans.xml and repack your jar, or manually set your packages to `ignore` on their CDI configuration.